test/AutoRest.TestServer.Tests/Common/Output/Models/Requests/PagingMethodTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+
+using System;
+using NUnit.Framework;
+using AutoRest.CSharp.Output.Models.Shared;
+
+namespace AutoRest.CSharp.Output.Models.Requests.Tests
+{
+    public class PagingMethodTests
```

Comment:
For each mgmt test project we should have a class that inherits from TestProjectsTests and OutputLibraryTests to get basic validation on all of our test projects